explore/src/main/scala/explore/targeteditor/ElevationPlotSemester.scala

```diff
@@ -57,111 +58,108 @@ object ElevationPlotSemester:
       .useContext(AppContext.ctx)
       .useResizeDetector()
       .useSerialState(none[Chart_])
-      .useEffectWithDepsBy((props, _, _, chartOpt) => chartOpt.value.isDefined)(
-        (props, ctx, _, chartOpt) =>
-          _ =>
-            chartOpt.value.value
-              .map { chart =>
-                import ctx.given
+      .useEffectWithDepsBy((props, _, _, chartOpt) =>
```

Comment:
The changes in this file did not seem to affect this problem, but it seems more correct to me to have the effect depend on the values that it uses. Using `Hide whitespace` helps show what actually changed.